src/scheduler/unpacking_scheduler.py

```diff
@@ -78,7 +100,7 @@ def shutdown(self):
         self.in_queue.close()
         stop_processes(
             [self.work_load_process, self.extraction_process],
-            10,  # give containers enough time to shut down
+            THROTTLE_INTERVAL,
```

Comment:
The `10` was meant to give the containers time to shut down (just as the comment said). What you may not have noticed is, that the containers are actually not stopped here but rather a couple of lines below (which means all the waiting is not necessary and `THROTTLE_INTERVAL` was the larger wait interval of both worker loops). Normally, this wouldn't be a problem because the process will join and we don't wait. But for whatever reason some thread(s) in the `extraction_loop` don't finish and the process needs to be killed (it seems not to be one of the extractor threads -- I spent some time debugging that and they are all not `is_alive()`).